types/src/system_txn.rs

```diff
@@ -15,3 +15,15 @@ pub enum SystemTransaction {
 pub struct DummySystemTransaction {
     pub nonce: u64,
 }
+
+impl SystemTransaction {
+    pub fn dummy(nonce: u64) -> Self {
+        Self::DummyTopic(DummySystemTransaction { nonce })
```

Comment:
can we make this dummy topic test only?

Reply:
It seem at least 1 variant is needed when `impl Arbitrary for SystemTransaction`?